lab-beeksma/index.js

```diff
@@ -0,0 +1,8 @@
+
+const arr = require('./lib/fp.js');
+
+var args = arr.nMap(process.argv, function(x){
```

Comment:
Typically "command line arguments" excludes the first two elements of `argv`, which represent the Node executable and script file.